lib/access_token_store.php

```diff
@@ -8,7 +8,7 @@ class AccessTokenStore
     private $refreshToken;
     private $refreshTokenExpiresAt;
 
-    const FILE_NAME = "config/access_token.yml";
+    const FILE_NAME = "db/access_token.yml";
```

Comment:
Please put a `.htaccess` file into the `db/` folder.

Reply:
Addressed